cpmpy/solvers/exact.py

```diff
@@ -167,10 +166,12 @@ def solve(self, time_limit=None, assumptions=None, **kwargs):
                 - False     if no solution is found
         """
         from exact import Exact as xct
-
+        
         # set additional keyword arguments
         if(len(kwargs.items())>0):
-            warnings.warn(f"Exact only supports options at initialization: {kwargs.items()}")
+            wrn_txt = f"Exact only supports options at initialization. Ignoring additional options {kwargs.items()}\n"
+            wrn_txt += "Use cp.SolverLookup.lookup('exact', **{parameter-with-hyphen: 42}) to set Exact parameters"
```

Comment:
seems unusual to do a string addition computation for the printing of a warning...

don't we have a more standard way of doing a multi-line warning? if not OK, but it looks unusual to me

Reply:
It is because of the curly brackets in the second line of tht warning text that I had to split it; otherwise I cannot format the first part to kwargs.items()